packages/contentful--create-contentful-app/src/types.ts

```diff
@@ -5,9 +5,10 @@ export type CLIOptions = Partial<{
   typescript: boolean;
   source: string;
   example: string;
-}>
+  action: boolean;
+}>;
 
 export const ContentfulExample = {
```

Comment:
Probably this is outdated and we need to update this variable and the functions relying on it. And have a single source of truth of what 'template' is. But that's for another PR.